wiki/Mascots/en.md

```diff
@@ -14,11 +14,11 @@ pippi, stylized with a lowercase "p", is the osu!standard mascot that joined on
 
 ### Yuzu
 
-<!-- *For the news post, see: [Meet Yuzu](/home/news/89483664163).* -->
+*For the news post, see: [Meet Yuzu](/home/news/2014-06-21-meet-yuzu).*
```

Comment:
Fix the other updated translations(only one I think) along with the joined date change.